import unittest
from venture.ripl import utils
from venture.exception import VentureException

class TestRiplUtils(unittest.TestCase):

    pass




if __name__ == '__main__':
    unittest.main()
